### PR TITLE
Update sendmail-monitoring-integration.mdx

### PR DESCRIPTION
**Pull request by pkosanam**
Ticket: https://issues.newrelic.com/browse/NR-167414

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx [hunk 1]
```diff
@@ -40,7 +40,7 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
   /etc/newrelic-infra/integrations.d
   ```
 
-2. Use the below snippet to update your configuration file named `sendmail-flex-config.yml`.
+2. Update your configuration file `sendmail-flex-config.yml` using the following snippet:
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:43]: @pkosanam Your original version said "Below are the fields which are to be updated on the below snippet." 

Which fields are they supposed to update?

Comment (pkosanam) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:43]: hi @ally-sassman , user has to update the value "FAILED_MESSAGES_FILE_NAME" present in the snippet, else there will be error recorded on integration.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx [hunk 2]
```diff
@@ -40,138 +40,141 @@ Flex comes bundled with the New Relic infrastructure agent. To create a flex con
...
+
+Kindly update the below value present in the snippet:
+`FAILED_MESSAGES_FILE_NAME`: Go to the path `/var/mail/` and check the file that has been created for failed messages
```
Comment (ally-sassman) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:46]:
```suggestion
2. Update your configuration file `sendmail-flex-config.yml` using the following snippet. 

Make sure to replace "FAILED_MESSAGES_FILE_NAME" references with your file name. To find this file name, go to `/var/mail/` and check the file that has been created for failed messages.
```

@pkosanam How does this look?

Comment (pkosanam) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration.mdx:46]: Thank you @ally-sassman, this looks good to me.. 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**